pkgs/development/libraries/libtiger/default.nix

```
@@ -9,7 +11,15 @@ stdenv.mkDerivation rec {
     sha256 = "0rj1bmr9kngrgbxrjbn4f4f9pww0wmf6viflinq7ava7zdav4hkk";
   };
 
-  nativeBuildInputs = [ pkg-config ];
+  patches = [
+    ./pkg-config.patch
+  ];
+
+  postPatch = ''
+    substituteInPlace configure.ac --replace "-Werror" "-Wno-error"
+  '';
```

Not sure if this works but potentially more robust since it does not require patching.

```suggestion
  NIX_FLAGS_COMPILE = ["-Wno-error"];
```

---

It would not, because this flag needs to be passed to automake, not the C compiler.